code/API_definitions/check-device-connectivity.yaml

```diff
@@ -99,10 +99,16 @@ components:
           $ref: "#/components/schemas/ConnectivityEventType"
         eventStatus:
           $ref: "#/components/schemas/EventStatusType"
+        countryCode:
+          $ref: "#/components/schemas/CountryCode"
+        countryName:
+          $ref: "#/components/schemas/CountryName"  
       required:
         - ueId
         - eventType
         - eventStatus
+        - countryCode
+        - countryName
```

Comment:
`countryCode` and `countryName` should not be `required`. Even if we agree that the home `countryCode` and `countryName` can be returned when the device is not roaming (this information can probably be deduced from `ueId` anyway), we need to consider user consent.

The user may consent to their roaming status being made available to the API caller, but not the country in which they are roaming. Providing the roaming country provides much more information on the device location than just knowing that the device is not in the home country.

Reply:
> we need to consider user consent.
> The user may consent to their roaming status being made available to the API caller, but not the country in which they are roaming. Providing the roaming country provides much more information on the device location than just knowing that the device is not in the home country.


You're right about user consent. How are we going to manage and seperate these cases? By having different scopes?